astropy/stats/histogram.py

```diff
@@ -23,13 +23,13 @@ def calculate_bin_edges(a, bins=10, range=None, weights=None):
     a : array_like
         Input data. The bin edges are calculated over the flattened array.
 
-    bins : int or list or str (optional)
+    bins : int or list or str, optional
```

Comment:
Again, to be consistent.
```suggestion
    bins : int, list, or str, optional
```